health/health.d/tcp_resets.conf

```diff
@@ -23,7 +23,10 @@
     warn: $this > ((($1m_ipv4_tcp_resets_sent < 5)?(5):($1m_ipv4_tcp_resets_sent)) * (($status >= $WARNING)  ? (1) : (20)))
    delay: up 20s down 60m multiplier 1.2 max 2h
  options: no-clear-notification
-    info: average TCP RESETS this host is sending, over the last 10 seconds (this can be an indication that a port scan is made, or that a service running on this host has crashed; clear notification for this alarm will not be sent)
+    info: average number of sent TCP RESETS over the last 10 seconds \
+          (this can be an indication that a port scan is made, \
+          or that a service running on this host has crashed; \
+          clear notification for this alarm will not be sent)
```

Comment:
```suggestion
    info: average number of sent TCP RESETS over the last 10 seconds. \
          This can be indicate a port scan, \
          or that a service running on this host has crashed. \
          Netdata will not send a clear notification for this alarm.
```